specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/Databases.json

```diff
@@ -481,47 +492,37 @@
         ],
         "responses": {
           "200": {
-            "description": "Successfully paused the database.",
-            "schema": {
-              "$ref": "#/definitions/Database"
-            }
+            "description": "Successfully completed database failover."
           },
           "default": {
-            "description": "*** Error Responses: ***\n\n * 400 CannotDeactivateWhenDeactivatingInProgress - Deactivation workflow failed because there is a deactivate workflow already running.\n\n * 400 FeatureDisabledOnSelectedEdition - User attempted to use a feature which is disabled on current database edition.\n\n * 404 SourceDatabaseNotFound - The source database does not exist.\n\n * 500 ActivateOrDeactivateWorkflowThrottling - Activation or deactivation workflow failed because there are too many concurrent workflows"
+            "description": "*** Error Responses: ***\n\n * 400 ManagementServiceFeatureDisabled - User attempted to use a feature which is disabled.\n\n * 400 DatabaseFailoverThrottled - There was a recent failover on the database or pool if database belongs in an elastic pool.\n\n * 400 DatabaseFailoverNotSupportedOnSKU - This type of customer initiated failover is not supported on the given SKU.\n\n * 409 ManagementServiceDatabaseBusy - Database '{0}' is busy with another operation. Please try your operation later.\n\n * 409 DatabaseNotInStateToFailover - The database is currently in a state such that failover cannot be issued."
           },
           "202": {
-            "description": "Pausing the database is in progress."
+            "description": "Database failover is in progress."
           }
         },
         "x-ms-long-running-operation": true,
         "x-ms-examples": {
-          "Pauses a database.": {
-            "$ref": "./examples/PauseDatabase.json"
+          "Failover an database": {
+            "$ref": "./examples/FailoverDatabase.json"
```

Comment:
I don't see the renamed file in this PR, is it missing?

Reply:
same comment as I replied in Line 472.